app/models/manageiq/providers/redhat/infra_manager/refresh_parser.rb

```diff
@@ -658,4 +659,10 @@ def self.datacenter_inv_to_hashes(inv, cluster_uids, vm_uids, storage_uids, host
 
     result
   end
+
+  private_class_method
```

Comment:
> So I think making this change is one step in a better direction, not perfect just better.

Yeah, valid point. I like the incremental step :)
But then `private_class_method` is used different than `private`. You have to give it a symbol with the method name